### PR TITLE
4.8 RNs: Add section in RNs explaining asynch add-on support status

### PR DESCRIPTION
**Pull request by jeana-redhat**
For https://github.com/openshift/openshift-docs/issues/29652#issuecomment-862388355

Preview: https://deploy-preview-33609--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-add-on-support-status

### CODE REVIEW

**Review by rolfedh — APPROVED** [review]
/lgtm

**Comment by jeana-redhat** [comment]
@SNiemann15 PTAL to see if you're on-board with the changes

**Comment by jeana-redhat** [comment]
@reestr FYI this is the PR related to the Slack thread where we are discussing layered and dependant components support/compatibility

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -32,6 +32,11 @@ Because only {op-system-base} 7.9 or later is supported for compute machines, yo
 
 With the release of {product-title} 4.8, version 4.5 is now end of life. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShift Container Platform Life Cycle Policy].
 
+[id="ocp-4-8-add-on-support-status"]
+== {product-title} layered and dependant component support and compatibility
+
+The scope of support for layered and dependant components of {product-title} changes independently of the {product-title} version. To determine the current support status and compatibility for an add-on, refer to its release notes. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShift Container Platform Life Cycle Policy].
```
Comment (mburke5678) [inline release_notes/ocp-4-8-release-notes.adoc:38]: s/dependant/dependent ?

**Comment by mburke5678** [comment]
@jeana-redhat One nit. Otherwise LGTM

**Pull request merged**